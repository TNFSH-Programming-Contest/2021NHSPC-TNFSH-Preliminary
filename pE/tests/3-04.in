243 500
3 4 6
5 3 13
4 6 21
5 4 26
6 7 28
5 7 14
5 7 6
5 7 24
8 7 27
8 5 18
8 7 6
8 7 25
8 7 13
8 7 14
8 7 17
8 7 29
8 7 24
8 7 5
7 9 13
8 7 26
9 10 27
8 10 9
7 10 21
11 12 25
8 11 21
12 13 7
8 11 16
14 13 19
11 14 12
15 11 21
11 13 28
10 16 25
8 11 30
11 13 30
13 17 28
8 11 23
17 18 9
8 11 6
13 18 20
8 11 29
8 19 27
19 16 8
8 15 19
8 19 16
19 16 16
20 18 25
16 11 16
20 18 25
20 18 17
21 18 21
20 18 19
20 8 20
20 18 18
20 21 11
20 18 26
20 18 19
18 22 6
23 24 29
20 22 25
20 22 7
25 26 16
24 25 23
22 23 29
20 22 20
23 26 24
22 26 29
22 26 9
27 28 22
26 27 19
29 20 19
29 28 6
30 31 9
31 32 8
32 33 25
30 29 10
30 33 13
30 34 8
33 28 16
33 28 9
30 28 20
30 28 18
35 36 30
34 28 25
35 37 9
38 39 10
40 39 9
40 38 20
30 28 24
35 41 13
40 39 5
40 39 15
41 36 16
42 43 30
44 45 5
37 36 25
45 46 26
47 44 5
47 44 7
30 28 27
46 28 14
48 49 14
40 39 19
50 48 11
36 51 25
30 28 24
50 49 27
30 47 23
28 35 22
52 42 6
40 39 12
53 52 7
53 52 23
40 39 5
52 43 21
40 39 13
39 54 26
55 53 25
51 56 21
54 57 15
58 49 13
59 40 9
40 39 20
59 40 13
30 28 10
58 49 16
50 49 6
43 59 23
55 43 29
56 60 29
28 60 23
61 62 19
63 62 25
60 64 13
58 49 10
65 66 18
67 63 18
68 57 15
50 65 28
68 69 20
69 57 14
50 58 19
68 55 13
64 70 12
70 71 5
68 57 13
68 57 26
50 71 8
66 30 18
71 49 10
50 71 8
72 61 19
73 74 12
72 67 27
72 62 8
74 75 7
72 62 14
72 76 14
75 68 25
68 50 29
76 62 7
49 57 20
75 68 12
77 62 21
57 78 25
78 79 9
80 73 17
77 62 30
72 62 20
72 77 21
81 82 14
79 83 8
83 81 5
84 72 13
85 86 28
87 85 28
81 82 24
84 72 17
82 87 6
88 80 9
88 89 6
86 89 26
90 91 22
80 89 29
92 90 12
62 93 24
94 88 19
93 95 18
96 97 22
84 98 16
98 93 30
94 92 13
88 89 29
99 96 27
100 101 28
101 95 5
94 88 17
97 102 14
99 103 22
84 100 16
102 104 6
84 95 26
99 102 17
105 91 30
89 91 19
84 100 18
94 91 28
94 105 29
84 95 5
99 103 27
103 94 12
94 91 19
99 104 17
104 84 29
95 91 12
99 104 19
106 91 27
106 91 28
106 91 25
106 91 24
106 91 14
106 91 25
106 91 18
106 99 13
107 91 28
106 91 6
106 107 24
106 91 28
106 91 5
106 91 13
91 108 17
106 108 26
106 108 17
109 108 13
106 108 17
109 106 13
109 108 17
110 111 16
111 112 12
113 114 29
115 108 30
116 115 14
117 118 18
119 110 5
109 116 26
115 108 6
120 121 18
113 114 5
121 122 19
108 123 12
124 125 19
124 117 11
118 109 22
123 125 17
125 120 13
119 112 10
126 113 7
113 114 13
126 114 19
127 128 12
126 114 26
129 130 14
124 122 22
124 122 17
131 132 9
129 127 22
129 128 11
133 134 28
135 132 23
129 128 16
136 131 16
114 133 28
134 137 21
135 136 19
112 138 17
126 139 21
130 140 8
130 140 28
141 122 13
142 119 27
129 130 30
139 133 12
143 142 15
130 140 21
122 144 29
145 140 7
131 132 29
122 144 6
135 132 6
129 145 22
144 146 26
141 124 24
129 128 18
126 133 5
129 128 16
129 128 15
140 128 17
146 132 30
129 140 16
147 141 9
144 148 27
129 149 20
137 147 8
129 143 30
129 128 30
129 149 30
149 128 27
147 144 14
147 144 17
144 135 30
150 126 19
137 147 22
138 151 22
152 151 21
129 151 9
150 137 10
147 144 19
129 152 19
129 138 12
148 135 14
153 154 18
155 154 23
156 153 22
153 155 30
151 154 10
150 144 7
128 154 26
156 153 8
156 153 23
132 157 5
158 156 6
129 154 8
157 159 27
160 129 13
160 129 7
157 159 18
159 161 12
162 163 9
144 157 27
164 160 30
161 165 6
158 154 24
144 165 15
158 154 11
150 144 9
166 154 8
164 154 28
166 167 24
166 167 13
166 158 30
168 154 20
167 154 15
163 169 8
170 171 29
166 168 18
172 173 29
174 162 29
166 150 27
165 164 28
170 171 21
166 154 27
175 172 30
176 177 18
178 166 7
179 178 17
166 154 21
170 179 30
169 176 19
166 154 15
180 173 6
171 174 10
177 175 25
154 171 25
181 182 19
171 180 20
183 181 24
184 185 11
186 187 27
182 187 18
185 186 23
186 183 7
183 187 23
173 184 10
170 171 14
170 187 21
187 188 29
170 188 25
189 170 20
190 189 21
190 188 5
191 190 16
192 193 23
193 191 26
194 195 22
192 194 10
192 195 8
192 195 28
188 195 27
196 192 18
195 197 18
196 192 25
198 196 8
198 196 7
197 199 10
196 197 10
197 199 19
197 199 28
196 197 7
200 201 20
200 201 21
202 200 15
200 201 8
203 204 5
205 206 7
198 207 7
202 201 14
199 207 21
205 198 28
208 202 17
209 208 23
210 209 19
208 202 21
205 206 27
211 212 28
205 206 25
213 211 18
212 214 22
205 206 29
201 203 23
205 207 30
215 216 26
210 209 12
217 218 23
219 210 13
218 205 6
219 209 17
217 213 25
220 204 18
214 215 9
216 221 25
222 219 12
217 205 26
222 219 6
207 222 19
219 209 11
207 209 15
223 217 6
205 206 16
205 209 11
220 204 9
224 209 17
206 224 27
223 217 10
217 205 9
209 204 29
221 220 15
223 217 13
217 205 28
225 226 18
223 227 5
225 226 29
205 204 24
228 229 5
225 226 24
217 204 14
227 217 13
204 230 9
223 217 22
231 232 21
231 232 9
223 230 18
233 234 20
223 230 9
223 230 20
233 234 28
226 235 19
228 231 16
236 223 30
235 236 6
233 234 28
233 230 16
233 225 17
228 229 25
233 230 28
234 230 30
233 230 7
237 238 25
238 233 10
232 237 29
229 230 10
239 230 25
228 239 5
240 241 23
240 241 29
230 242 19
242 240 27
243 2 14
241 243 19
228 2 23
228 2 24
1 228 28
