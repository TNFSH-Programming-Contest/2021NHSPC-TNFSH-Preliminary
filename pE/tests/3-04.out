38.51598843
